tests/storage/storage_migration/test_mtc_storage_class_migration.py

```diff
@@ -208,3 +210,48 @@ def test_migrate_vm_with_hotplugged_volume_after_storage_migration(
             except Exception as migration_exception:
                 vms_failed_migration[vm.name] = migration_exception
         assert not vms_failed_migration, f"Failed VM migrations: {vms_failed_migration}"
+
+
+@pytest.mark.parametrize(
+    "vms_for_storage_class_migration",
+    [
+        pytest.param(
+            {"vms_fixtures": ["windows_vm_with_vtpm_for_storage_migration"]},
+            id="mig_win_vm_with_vtpm",
+        )
+    ],
+    indirect=True,
+)
+@pytest.mark.tier3
+class TestStorageClassMigrationWindowsWithVTPM:
+    @pytest.mark.dependency(name=f"{TESTS_CLASS_NAME_WINDOWS}::test_vm_storage_class_migration_windows_vm_with_vtpm")
+    @pytest.mark.parametrize(
+        "source_storage_class, target_storage_class, online_vms_for_storage_class_migration",
+        [
+            pytest.param(
+                {"source_storage_class": py_config[STORAGE_CLASS_A]},
+                {"target_storage_class": py_config[STORAGE_CLASS_B]},
+                {"online_vm": [True]},  # Desired VM Running status for VMs in "vms_fixtures" list
+                marks=pytest.mark.polarion("CNV-11499"),
+                id="storage_migration_a_to_b_win_vm_with_vtpm",
+            )
+        ],
+        indirect=True,
+    )
+    def test_vm_storage_class_migration_windows_vm_with_vtpm(
+        self,
+        source_storage_class,
+        target_storage_class,
+        online_vms_for_storage_class_migration,
+        written_file_to_windows_vms_before_migration,
+        vms_boot_time_before_storage_migration,
+        storage_mig_plan,
+        storage_mig_migration,
+        deleted_old_dvs_of_online_vms,
+    ):
+        verify_windows_storage_migration_succeeded(
```

Comment:
as discussed today, there should be 3 tests here so it is:
- each test checks one thing
- it is clear what fails
- one verification does not block the other verifications,